src/hooks/useCryptoCurrencies.js

```diff
@@ -11,8 +12,15 @@ const useCryptocurrencies = ({
   onlyTerminated?: boolean,
   onlySupported?: boolean,
 }): CryptoCurrency[] => {
+  const [cryptos, setCryptos] = useState([])
```

Comment:
initial value should be the result of `listCryptoCurrencies` because we don't want to wait network OR in case something is bad with API